expression/builtin_time.go

```diff
@@ -4162,17 +4162,27 @@ func (c *unixTimestampFunctionClass) getFunction(ctx sessionctx.Context, args []
 // MySQL's Unix timestamp ranges in int32. Values out of range should be rewritten to 0.
```

Comment:
Maybe update this comment? Since we are not really returning a Timestamp, but a number with same base, but bigger range than the 32-bit timestamp.